packages/ui/src/Avatar/Avatar.tsx

```diff
@@ -3,26 +3,43 @@ import Image from 'next/image'
 import { type Author } from '@filecoin-foundation/utils/types/authorTypes'
 import { clsx } from 'clsx'
 
-const IMAGE_SIZE = 32
+const IMAGE_SIZE = {
+  default: 32,
+  large: 40,
+}
+
 const sharedAvatarStyle = 'rounded-full ring avatar-ring'
 
-export function Avatar({ firstName, lastName, image }: Author) {
+type AvatarProps = Author & {
+  size?: 'default' | 'large'
+}
+
+export function Avatar({
+  firstName,
+  lastName,
+  image,
+  size = 'default',
+}: AvatarProps) {
   if (image) {
     return (
       <Image
         src={image.src}
         alt={`Photo of ${firstName} ${lastName}`}
-        width={IMAGE_SIZE}
-        height={IMAGE_SIZE}
-        className={clsx(sharedAvatarStyle, 'inline-block size-8 object-cover')}
+        width={IMAGE_SIZE[size]}
+        height={IMAGE_SIZE[size]}
+        className={clsx(
+          sharedAvatarStyle,
+          'inline-block object-cover',
```

Comment:
why `inline-block`? 👀 both `AvatarGroup` and `AvatarGroupExpanded` are flex containers?